include/PointCloudTpl.h

```diff
@@ -37,12 +37,48 @@ namespace CCCoreLib
 			, m_currentOutScalarFieldIndex(-1)
 		{}
 
+		//! Copy Constructor
+		PointCloudTpl(const PointCloudTpl &rhs)
```

Comment:
What does rhs stands for?

Reply:
I would assume rhs == right hand side
i.e the value on the rhs side of the equal sign.

I've seen it used a lot in operator overloads and constructors over the years.